curriculum/utils.js

```diff
@@ -36,6 +36,29 @@ const superBlockToOrder = {
   'relational-database': 12
 };
 
+/**
+ * This order is used for i18n instances where a new certification is released
+ * from beta but is not audited, so cannot be reordered (due to the way we
+ * split the map)
+ */
+const superBlockNonAuditedOrder = {
```

Comment:
I'm not too picky on variable names, but I'm wondering if this is accurate and/or could be better. Maybe `rwdIsBetaSuperBlockOrder`. If it's functional, I really don't mind what we call it.

Reply:
Ended up having to move this anyway so I could use it in the tests

Reply:
Sounds good, I'm fine with it like this.